subsys/metrics/src/main/conf/conf.d/metrics.conf

```diff
@@ -1,10 +1,35 @@
 [metrics]
 enabled = false
-reporter = graphite.reporter
-reporter.enabled = false
+reporter.enabled = true
+
+######################################
+# List of enabled reporters (comma-separated)
+######################################
+
+reporter = graphite;zabbix
+
+############################################
+# GraphiteDB reporter options
+############################################
+
 graphite.reporter.hostname = localhost
 graphite.reporter.port = 2003
 graphite.reporter.prefix = local.one
 graphite.reporter.simple.period = 30
 graphite.reporter.jvm.period = 300
 graphite.reporter.healthcheck.period = 600
+
+############################################
+# Zabbix reporter options
+############################################
+
+zabbix.reporter.api.host.url = example.com
```

Comment:
I was hoping to get a bit more documentation in the configuration files about what each of these does, and to simplify these property names a bit more:

```
    -    zabbix.reporter.api.host.url = example.com
    +   # API URL used to access Zabbix from a web browser 
    +    zabbix.api.url=example.com

    -    zabbix.reporter.host.port = 10051
    +   # Target port for Zabbix reporting
    +    zabbix.port=10051

    -    zabbix.reporter.host = 192.168.1.1
    +   # Target host for Zabbix reporting
    +    zabbix.host=192.168.1.1

    -    zabbix.reporter.user = test
    +   # Zabbix username for reporting
    +   zabbix.user=test

    -    zabbix.reporter.pwd = test
    +   # Zabbix password for reporting
    +   zabbix.password=test

    -    zabbix.reporter.local.hostname = example.com
    +   # This is the hostname of the current Indy instance
    +   zabbix.indy.host=example.com

    -    zabbix.reporter.prefix = local.one
    +   # I don't actually know what this is for...
    +   zabbix.prefix=local.one
```

Basically, transform things like `zabbix.reporter.` to `zabbix.`, and `graphite.reporter.` to `graphite.`. Then, I think it would be useful to have some hint of what each setting does, even if it seems obvious.